File: python/program/Orang.py
from InformasiKontak import InformasiKontak

# subclass Orang
class Orang(InformasiKontak):
    
    def __init__(self, noHp: str, email: str, alamat: str,
                 nama: str, usia: int, tanggalLahir: str):
        # panggil constructor parent
        super().__init__(noHp, email, alamat)
        # atribut khusus Orang
        self._nama = str(nama)
        self._usia = int(usia)
        self._tanggalLahir = str(tanggalLahir)

    # Getter & Setter Nama
    def getNama(self) -> str:
        return self._nama
    
    def setNama(self, nama: str) -> None:
        self._nama = str(nama)

    # Getter & Setter Usia
    def getUsia(self) -> int:
        return self._usia
    
    def setUsia(self, usia: int) -> None:
        self._usia = int(usia)

    # Getter & Setter Tanggal Lahir
    def getTanggalLahir(self) -> str:
        return self._tanggalLahir
    
    def setTanggalLahir(self, tanggalLahir: str) -> None:
        self._tanggalLahir = str(tanggalLahir)

File: java/program/Orang.java
public class Orang extends InformasiKontak {
    /* Class Orang ini mewarisi (extends) dari class InformasiKontak, jadi otomatis punya atribut dari parent class.
     Di sini saya tambahkan atribut khusus untuk orang */
    private String nama;
    private int usia;
    private String tanggalLahir;

    // Constructor ini dipakai buat isi data awal.
    // Karena class ini extends InformasiKontak, kita panggil juga constructor dari parent pakai super().
    public Orang(String noHp, String email, String alamat,
                 String nama, int usia, String tanggalLahir) {
        super(noHp, email, alamat);
        this.nama = nama;
        this.usia = usia;
        this.tanggalLahir = tanggalLahir;
    }

    // Getter dan setter di bawah ini dipakai untuk ambil dan ubah nilai
    // dari atribut nama, usia, dan tanggal lahir.
    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public int getUsia() {
        return usia;
    }

    public void setUsia(int usia) {
        this.usia = usia;
    }

    public String getTanggalLahir() {
        return tanggalLahir;
    }

    public void setTanggalLahir(String tanggalLahir) {
        this.tanggalLahir = tanggalLahir;
    }
}
